.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
@aaronsteers was this meant to be added in this PR or https://github.com/MeltanoLabs/meltano-map-transform/pull/78?

Reply:
Was build on top of #78. See my note in description:

> Note:
> 
> This also builds on:
> 
> * [chore: Add devcontainer config #78](https://github.com/MeltanoLabs/meltano-map-transform/pull/78)
> 
> That one should probably merge first. If we are in a rush to merge, we can approve all at once or else rebase on main with the changes from #78.

Reply:
@aaronsteers cool, we'll wait for #78 to be merged then 👍